### ISSUE TITLE
user session broken in some rare cases

### ISSUE THREAD

**Issue by user2589** [mbi/django-rosetta#3]
To reproduce this issue:
- session backend is cached or cached_db,
- cache backend is memcached
- po file is 700kb or more

as lang_sel view stores po file in session, and memcached doesn't store values bigger than 1Mb, session is cut and some data may be lost

**Comment by mbi** [comment]
Yes, I reckon this could happen. I wouldn't recommend having a 700k pofile in the first place, but this could indeed happen. 
Any suggestions on how to handle this?

**Comment by user2589** [comment]
I'll make a pull request for fix soon
